Content.Server/StationEvents/MeteorSchedulerSystem.cs

```diff
@@ -13,12 +15,22 @@ namespace Content.Server.StationEvents;
 public sealed class MeteorSchedulerSystem : GameRuleSystem<MeteorSchedulerComponent>
 {
     [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+    private TimeSpan _meteorMinDelay;
+    private TimeSpan _meteorMaxDelay;
 
     protected override void Started(EntityUid uid, MeteorSchedulerComponent component, GameRuleComponent gameRule, GameRuleStartedEvent args)
     {
         base.Started(uid, component, gameRule, args);
 
-        component.NextSwarmTime = Timing.CurTime + RobustRandom.Next(component.MinSwarmDelay, component.MaxSwarmDelay);
+        _meteorMinDelay = TimeSpan.FromMinutes(_cfg.GetCVar(CCVars.MeteorSwarmMinTime));
+        _meteorMaxDelay = TimeSpan.FromMinutes(_cfg.GetCVar(CCVars.MeteorSwarmMaxTime));
+
+        _cfg.OnValueChanged(CCVars.MeteorSwarmMinTime, f => { _meteorMinDelay = TimeSpan.FromMinutes(f); });
+        _cfg.OnValueChanged(CCVars.MeteorSwarmMaxTime, f => { _meteorMaxDelay = TimeSpan.FromMinutes(f); });
```

Comment:
OnValueChanged has an invokeImmediately option so you don't have to double up like this.

Reply:
And done. Thanks for helping me make a better system!